File: spoj py/P146PROC.py
from sys import exit
s = raw_input()
s = s+" "
i = 0
k = 0
Color = range(0,4)
while k<len(s) and i<4:
    temp = ''
    while s[k]==' ':
        k = k+1
    while s[k]!=' ':
        temp = temp+s[k]
        k = k+1
    Color[i] = long(temp)
    i = i+1

del s
del i
del k
del temp

Count = 0

for i in range(0,3):
    for j in range(i+1,4):
        if Color[j]==Color[i]:
            Count = Count + 1
            break

print Count

exit(0)

File: P146PROC.cpp
#include<stdio.h>

int main(void)
{
	long Color[4];
	for(int i=0;i<4;i++)
		scanf("%ld",&Color[i]);
	int Count=0;
	for(int i=0;i<3;i++)
	{
		for(int j=i+1;j<4;j++)
		{
			if(Color[j]==Color[i])
			{
				Count++;
				break;
			}
		}
	}
	printf("%d",Count);
 return 0;
}

